resources/gcloud.rb

```diff
@@ -1,27 +1,107 @@
 resource_name :gcloud
 
-# property :version, String, default: nil
+property :version, String, default: ''
+property :gcloud_path, String, default: '/usr/bin/gcloud'
+property :gcloud_repo, String, default: '/etc/yum.repos.d/google-cloud-sdk.repo'
```

Comment:
I'm not sure  if we need "gcloud_repo" property for users to use, I don't see the use case for this.